### ISSUE TITLE
Include pre-compiled libraries at release

### ISSUE THREAD

**Issue by rnbguy** [informalsystems/modelator#161]
Setup a CI to provide pre-compiled library packages for MBT-Go (and future FFI).

The libraries should be compiled for,

- [ ] `x86_64-unknown-linux-gnu`
- [ ] `aarch64-unknown-linux-gnu`
- [ ] `x86_64-apple-darwin`
- [ ] `aarch64-apple-darwin`
- [ ] `x86_64-pc-windows-gnu`

**Comment by rnbguy** [comment]
The GH action matrix would look like,

```
- { os: ubuntu-latest, target: x86_64-unknown-linux-gnu }
- { os: ubuntu-latest, target: aarch64-unknown-linux-gnu }
- { os: macos-latest, target: x86_64-apple-darwin }
- { os: macos-latest, target: aarch64-apple-darwin }
- { os: windows-latest, target: x86_64-pc-windows-gnu }
```

**Comment by rnbguy** [comment]
Closes #155
